### PR TITLE
4390 - [UTCTimeFormat] Convert to ES6 Plugin

### PR DESCRIPTION
**Pull request by mariuszr**
Addresses #4390 

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [ ] Testing instructions included in associated issue?

### CODE REVIEW

**Comment by unlikelyzero** [comment]
@mariuszr for some reason, this PR is not being picked up by our codecoverage bot. I've issued a rebase to see if it triggers the change. I've created https://github.com/nasa/openmct/issues/4413 to investigate further. Sorry for the inconvenience!

**Comment by jvigliotta** [comment]
@mariuszr  this looks great! can you add test similar to: https://github.com/nasa/openmct/blob/master/src/plugins/ISOTimeFormat/pluginSpec.js then this should be good to go

**Comment by mariuszr** [comment]
> @mariuszr this looks great! can you add test similar to: https://github.com/nasa/openmct/blob/master/src/plugins/ISOTimeFormat/pluginSpec.js then this should be good to go. If not, we can always create an issue for a test. Let me know.

**Comment by mariuszr** [comment]
Reopen as accidentally I  closed it :)

**Comment by mariuszr** [comment]
Hi @jvigliotta, I have added a test for the plugin. Please check it and let me know

File: src/plugins/UTCTimeFormat/pluginSpec.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+import UTCTimeFormat from './UTCTimeFormat.js';
+
+describe('the plugin', () => {
+    const ISO_KEY = 'utc';
```
Comment (jvigliotta) [inline src/plugins/UTCTimeFormat/pluginSpec.js:26]: Can we change this to UTC_KEY?

**Review by jvigliotta — CHANGES_REQUESTED** [review]
One _small_ little change, then looks good to go. Thank you for this! I like the optional format checks in the tests as well, nice work!

File: src/plugins/UTCTimeFormat/pluginSpec.js (hunk 1 above)
Comment (mariuszr) [reply src/plugins/UTCTimeFormat/pluginSpec.js]: Oh, my bad. Corrected.

**Review by jvigliotta — APPROVED** [review]
Looks great, thanks @mariuszr !

### Reviewer Checklist
Check | Passed
-- | --
Changes appear to address issue?? | Y
Appropriate unit tests included? | Y
Code style and in-line documentation are appropriate? | Y
Commit messages meet standards? | Y
Has associated issue been labelled unverified? (if PR closes issue) | Y

**Pull request merged**